fastlane/lib/fastlane/actions/ensure_git_status_clean.rb

```diff
@@ -65,6 +71,12 @@ def self.available_options
                                        description: "The flag whether to show the git diff if the repo is dirty",
                                        optional: true,
                                        default_value: false,
+                                       is_string: false),
+          FastlaneCore::ConfigItem.new(key: :ignored,
+                                       env_name: "FL_ENSURE_GIT_STATUS_CLEAN_IGNORED_FILE",
+                                       description: "The flag whether to ignore file the git status if the repo is dirty",
+                                       optional: true,
+                                       default_value: false,
                                        is_string: false)
```

Comment:
`is_string` should be true, right? `--ignored` looks like its 3 strings values and not a boolean?

```sh
       --ignored[=<mode>]
           Show ignored files as well.
           The mode parameter is used to specify the handling of ignored files. It is optional: it defaults to traditional.
           The possible options are:
           o   traditional - Shows ignored files and directories, unless --untracked-files=all is specified, in which case individual files in ignored
               directories are displayed.
           o   no - Show no ignored files.
           o   matching - Shows ignored files and directories matching an ignore pattern.
           When matching mode is specified, paths that explicitly match an ignored pattern are shown. If a directory matches an ignore pattern, then it is
           shown, but not paths contained in the ignored directory. If a directory does not match an ignore pattern, but all contents are ignored, then the
           directory is not shown, but all contents are shown.
```

Reply:
Yes `is_string` to `true`